abci/proposal_auction.go

```diff
@@ -86,6 +86,12 @@ func (h *ProposalHandler) VerifyTOB(ctx sdk.Context, proposalTxs [][]byte) (*Auc
 		return nil, fmt.Errorf("failed to unmarshal last commit info from auction info: %w", err)
 	}
 
+	// verify that the included vote extensions are valid in accordance with the
+	// the preferences of the application
+	if err := h.validateVoteExtensionsFn(ctx, ctx.BlockHeight(), lastCommitInfo); err != nil {
```

Comment:
We shld think abt noting down our experiences w/ VEs at present, and thinking abt how the API can be changed to make the dev UX better